<launch>

  <node name="bessica_d_driver_node" pkg="bessica_d_driver" type="bessica_d_driver_node" output="screen">
  
    <!-- Serial Port Configuration -->
    <param name="port"           type="string" value="/dev/ttyCH341USB0"/>
    <param name="baud_rate"      type="int"    value="921600"/>
    
    <!-- Driver Behavior Configuration -->
    <param name="debug_mode"     type="bool"   value="true"/>
    <param name="rate_limit_sec" type="double" value="0.01"/>
    
  </node>

</launch>